examples/PCA.cpp

```diff
@@ -0,0 +1,48 @@
+// Include necessary header files
+#include <Matrix.hpp>
+#include <PCA.hpp>
+#include <matplotlibcpp.hpp>
```

Comment:
Is there any plot that can be added for visualization?

Reply:
Na